core/src/test/java/com/ericsson/bss/cassandra/ecchronos/core/AbstractCassandraTest.java

```diff
@@ -77,6 +77,11 @@ public boolean getRemoteRouting()
             {
                 return true;
             }
+
+            @Override
+            public String getSerialConsistency(){
+                return "DEFAULT";
```

Comment:
Why not use a regular constant for this? This kind of connects to the reasoning why to use proper constants; e.g. now you have multiple places that needs update if you need to change this.

Reply:
Done